strings/base_coroutine_threadpool.h

```diff
@@ -21,6 +21,13 @@ namespace winrt::impl
         return (0 == WINRT_CoGetApartmentType(&aptType, &aptTypeQualifier)) && ((aptType == 0 /*APTTYPE_STA*/) || (aptType == 3 /*APTTYPE_MAINSTA*/));
     }
 
+    inline bool is_mta() noexcept
+    {
+        int32_t aptType;
+        int32_t aptTypeQualifier;
+        return (0 == WINRT_CoGetApartmentType(&aptType, &aptTypeQualifier)) && ((aptType == 1 /*APTTYPE_MTA*/));
```

Comment:
Does it make sense to introduce an enum and have only one function to get the apartment type?

Reply:
No, the question is simpler. Does a call context need to be captured.